packaging/os/apt.py

```diff
@@ -755,6 +755,30 @@ def get_updated_cache_time():
     return mtimestamp, updated_cache_time
 
 
+# https://github.com/ansible/ansible-modules-core/issues/2951
+def get_cache(module):
+    '''Attempt to get the cache object and update till it works'''
+    cache = None
+    try:
+        cache = apt.Cache()
+    except SystemError as e:
```

Comment:
This exception syntax isn't compatible with python 2.4, which is why CI is failing.